$MirOS: ports/plan9/kencc/patches/patch-FreeBSD_386_include_lib9_h,v 1.1 2006/03/31 03:50:14 tg Exp $
--- FreeBSD/386/include/lib9.h.orig	Thu Mar 30 22:21:28 2006
+++ FreeBSD/386/include/lib9.h	Thu Mar 30 23:15:10 2006
@@ -418,9 +418,9 @@ struct Waitmsg
 	char	*msg;
 } Waitmsg;
 
-extern	void	_exits(char*);
+extern	void	_exits(char*) __attribute__((__noreturn__));
 
-extern	void	exits(char*);
+extern	void	exits(char*) __attribute__((__noreturn__));
 extern	int	create(char*, int, int);
 extern	int	errstr(char*, uint);
 
